### PR TITLE
migration to clean-up duplicate chunks

### PR DESCRIPTION
**Pull request by spolu**
This migrations attempt to retrieve and delete ducplicate chunks in our qdrant collections.

Tested locally successfully.

### CODE REVIEW

**Review by fontanierh — APPROVED** [review]

File: front/migrations/20240110_kill_duplicate_chunks.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    }
+    let offset:
+      | number
+      | undefined
+      | null
+      | string
+      | Record<string, unknown> = 0;
+    while (offset !== null && offset !== undefined) {
+      const res = await client.scroll(c.name, {
+        // with_payload: true,
```
Comment (philipperolet) [inline front/migrations/20240110_kill_duplicate_chunks.ts:35]: did you comment just for testing and will re-enable later? (IMHO param necessary)

Comment (philipperolet) [reply front/migrations/20240110_kill_duplicate_chunks.ts:35]: might also benefit from  `with_vector: false` (doc unclear on the default)

**Review by philipperolet — COMMENTED** [review]
LGTM (modulo comment check)

File: front/migrations/20240110_kill_duplicate_chunks.ts (hunk 1 above)
Comment (spolu) [reply front/migrations/20240110_kill_duplicate_chunks.ts:35]: We need neither of them :+1:

**Comment by spolu** [comment]
Late merge for the record

**Pull request merged**